Source/WebCore/dom/EventListenerMap.h

```diff
@@ -74,6 +74,16 @@ class EventListenerMap {
             callback(entry.first, entry.second.size());
     }
 
+    template <typename CallbackType>
```

Comment:
I think we usually omit the space?

Reply:
Hm... we seem to mix both styles. 907 with a space and 4089 without a space. I guess without a space is more common so I'd align with that.